Recursive/Factorial.js

```diff
@@ -9,6 +9,18 @@
  */
 
 const factorial = (n) => {
+  if (typeof n !== 'number') {
+    throw new TypeError('Not a Number')
+  }
+
+  if ((n % 1) !== 0) {
```

Comment:
Maybe use `Number.IsInteger(n)` here? Same functionality but more readable imo.

Reply:
Done.
But It turns out that, checking `typeof n !== 'number'` is not necessary when `Number.isInteger(n)` is used.

Reply:
That's true too, so simpler code.

Reply:
Okay.